### PR TITLE
Introduce annotations

### PR DESCRIPTION
**Pull request by huntc**
Annotations are supported by the forthcoming ConductR 2.1.

### CODE REVIEW

**Comment by typesafe-tools** [comment]
[Update sbt-conductr to include annotations](https://trello.com/c/7VzTJvxK/637-update-sbt-conductr-to-include-annotations)

File: README.md [hunk 1]
```diff
@@ -392,6 +392,7 @@ The following bundle settings are provided under the `BundleKeys` object:
 
 Name                  | Description
 ----------------------|-------------
+annotations           | An optional HOCON string representing additional metadata that you may wish to associate with a bundle. Key names should be in accordance with the [OCI image annotation](https://github.com/opencontainers/image-spec/blob/master/annotations.md) conventions.
```
Comment (markusjura) [inline README.md]: I'd add an annotations example, e.g. 

```
{
  reference-id = "124f534as",
  zones = ["us-east", "us-west"]
}
```

Users might not familiar with the HOCON syntax, especially that they need to wrap the key value pairs in `{}`.

File: src/main/scala/com/lightbend/conductr/sbt/BundleImport.scala [hunk 2]
```diff
@@ -254,6 +254,11 @@ object BundleImport {
       """An array of strings that can be used to further qualify a bundle name. Just as with a name, these strings are intended for human consumption and ConductR makes no assumptions about their value - see "compatibilityVersion" for semantically relevant versioning. Tags are often represented as versions e.g. "1.0.0-beta.1", "version1"" etc. By default we use the project version."""
     )
 
+    val annotations = SettingKey[Option[String]](
+      "bundle-annotations",
+      """A HOCON string representing additional metadata that you may wish to associate with a bundle. Key names should be in accordance with the OCI image annotation conventions: https://github.com/opencontainers/image-spec/blob/master/annotations.md."""
```
Comment (markusjura) [inline src/main/scala/com/lightbend/conductr/sbt/BundleImport.scala]: You can also mention that it defaults to `None`.

File: README.md (hunk 1 above)
Comment (markusjura) [inline README.md]: You can also mention that it defaults to `None`.

File: src/main/scala/com/lightbend/conductr/sbt/BundleImport.scala (hunk 2 above)
Comment (huntc) [inline src/main/scala/com/lightbend/conductr/sbt/BundleImport.scala]: Done

**Comment by huntc** [comment]
jenkins

**Pull request merged**